test/e2e-test-application/src/app/project/settings/settings.component.html

```diff
@@ -91,5 +91,21 @@ <h3 class="fd-layout-panel__title">
         to go to <i>project two</i>, where you can try out hasBack functionality
       </div>
     </div>
+    <div class="fd-layout-panel">
```

Comment:
```suggestion
    <div *ngIf="isModal" class="fd-layout-panel">
```
It would be good to hide the whole panel if **isModal** is not set, since at the current state it distorts the non-modal settings page: 
![Screenshot 2020-08-25 at 16 09 43](https://user-images.githubusercontent.com/18335489/91184986-8dde1180-e6ed-11ea-9d3b-588bb02be7f8.png)